csharp/test/Vista.SDK.Tests/LocalIdTests.cs

```diff
@@ -106,6 +106,46 @@ public void Test_LocalId_Build_Valid(Input input, string expectedOutput)
         Assert.Equal(expectedOutput, localIdStr);
     }
 
+    [Fact]
+    public void Test_LocalId_Build_AllWithout()
+    {
+        var (_, vis) = VISTests.GetVis();
+
+        var visVersion = VisVersion.v3_4a;
+
+        var gmod = vis.GetGmod(visVersion);
+        var codebooks = vis.GetCodebooks(visVersion);
+
+        var primaryItem = gmod.ParsePath("411.1/C101.31-2");
+        var secondaryItem = gmod.ParsePath("411.1/C101.31-5");
+
+        var localId = LocalIdBuilder
+            .Create(visVersion)
+            .WithPrimaryItem(primaryItem)
+            .TryWithSecondaryItem(secondaryItem)
+            .WithVerboseMode(true)
+            .TryWithMetadataTag(codebooks.TryCreateTag(CodebookName.Quantity, "quantity"))
+            .TryWithMetadataTag(codebooks.TryCreateTag(CodebookName.Content, "content"))
+            .TryWithMetadataTag(codebooks.TryCreateTag(CodebookName.Position, "position"))
+            .TryWithMetadataTag(codebooks.CreateTag(CodebookName.State, "state"))
+            .TryWithMetadataTag(codebooks.CreateTag(CodebookName.Content, "content"))
+            .TryWithMetadataTag(codebooks.CreateTag(CodebookName.Calculation, "calculate"));
+
+        Assert.True(localId.IsValid);
+
+        var allWithout = localId
```

Comment:
Veldig nice, men synes du det funka greit med withoutMetadataTag eller tror du withoutQuantity etc hadde vært lettere? Da slipper man å sende med MetadataTag. Ser det er veldig tungvint. Evt bør withoutMetadataTag skriver om til å ta in CodebookName som inputParam synes jeg :)